static/js/invite.js

```diff
@@ -123,7 +123,7 @@ function generate_multiuse_invite() {
         beforeSend,
         success(data) {
             ui_report.success(
-                i18n.t('Invitation link: <a href="__link__">__link__</a>', {
+                i18n.t('Invitation link: <a href="__link__" target="_blank">__link__</a>', {
```

Comment:
Links using `target=_blank" should use rel="nopener noreferrer"; grep the codebase for examples of how to do this correctly.  I also think we should move the HTML out of the translation tag, so it's just 

i18n.t("Invitation link: __link__", {link: "<a ..."}), or perhaps better, just move this little block of rendering to a tiny handlebars template. (We generally prefer to not write out HTML in JavaScript code).

Reply:
I've made a new `invite_multiuse_link.hbs` file for this.